src/OVAL/oval_component.c

```diff
@@ -868,29 +868,29 @@ void oval_component_free(struct oval_component *component)
 	case OVAL_COMPONENT_OBJECTREF:{
 			oval_component_OBJECTREF_t *objectref = (oval_component_OBJECTREF_t *) component;
 			if (objectref->item_field != NULL)
```

Comment:
I think this condition can be removed. But there will be more of them I guess. It isn't necessary to fix them.